apollo-router/tests/snapshots/set_context__set_context_union_rust_qp.snap

```diff
@@ -47,15 +47,6 @@ expression: response
                       "scopes": []
                     },
                     "contextRewrites": [
-                      {
```

Comment:
Bugs:
- This snapshot differs from `apollo-router/tests/snapshots/set_context__set_context_union.snap`, and this PR change brings them back into alignment. Not sure why we allowed the difference there.